composer.json

```diff
@@ -185,6 +185,7 @@
         "nikic/php-parser":"^4.7",
         "doctrine/inflector": "^1.4",
         "sylius/grid-bundle": "1.7.4",
+        "symfony/doctrine-bridge": "4.4.16",
```

Comment:
This has to be fixed to `4.4.15` to make it work. 

Note that in #11970  it says

> Steps to reproduce
> Update symfony/doctrine-bridge from 4.4.15 to 4.4.16 and run bin/console cache:clear

/cc @lchrusciel

Reply:
I hope since v4.4.17 it will be fixed. However, maybe we should add `^4.4.16`, and put the upper constraint once we will be sure it works once again.